src/__tests__/Feedback.test.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react'
+import { render, screen, userEvent } from '../tests/testUtils'
+import faker from 'faker'
+import App from '../App'
+import {
+  fetchedCollectionData,
+  fetchedDocumentData,
+} from '../tests/data/raidsData'
+
+const buildFeedback = {
+  name: faker.name.firstName(),
+  repoName: faker.commerce.productName(),
+  description: faker.lorem.paragraph(10),
+  rating: Math.round(faker.random.number({ min: 1, max: 5 })),
+}
+
+beforeAll(() => {
+  jest.doMock('../utils/useDocument', () =>
+    jest.fn().mockReturnValue(fetchedDocumentData),
+  )
+  jest.doMock('../utils/useCollection', () =>
+    jest.fn().mockReturnValue(fetchedCollectionData),
+  )
+  jest.spyOn(console, 'log')
```

Comment:
When we use the emulators, we don't have to mock anything because we are just interfacing with the emulator instead of the real thing - yes you still have to "seed" the data.

This can be done by just importing from the `devData` dir for now, as the website is really read-only.

Reply:
> When we use the emulators, we don't have to mock anything

totally agree. i tried it before mocking but when it failed i when with mocking. i will do remove the mocks.